Description: Upstream changes introduced in version 2.3.0+r5065-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 daq-midas (2.3.0+r5065-1) unstable; urgency=low
 .
   * Initial release.
   * midas: 2.3.0, subversion r5065
   * mxml: subversion r68
   * Add autoconf support to midas sources.
 .
 The person named in the Author field signed this changelog entry.
Author: Exaos Lee <exaos.lee@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ daq-midas-2.3.0+r5065/.gitmodules
@@ -0,0 +1,6 @@
+[submodule "midas"]
+	path = midas
+	url = git@memes:bot-midas.git
+[submodule "mxml"]
+	path = mxml
+	url = git@memes:bot-mxml.git
